issue_12/script.js

```diff
@@ -0,0 +1,75 @@
+function startLoading() {
+    const loadingImg = document.createElement('img');
+    loadingImg.src = 'loading-circle.gif';
+    loadingImg.alt = 'ローディング画像';
+    document.getElementById('js-loading').appendChild(loadingImg);
+}
+
+function removeLoading() {
+    document.getElementById('js-loading').remove();
+}
+
+function createLists(resolvedData) {
+    const ul = document.getElementById('js-ul');
+    const fragment = document.createDocumentFragment();
+    for (const item of resolvedData) {
+        const li = document.createElement('li');
+        const a = document.createElement('a');
+        const img = document.createElement('img');
+
+        a.href = item.to;
+        a.textContent = item.text;
+        img.src = item.img;
+        img.alt = item.alt;
```

Comment:
MUST
The results are as follows.
```
<li><a href="undefined">ブックマーク<img src="img/1.png" alt="画像１"></a></li>
<li><a href="undefined">メッセージ<img src="img/2.png" alt="画像２"></a></li>
```
There is no href. Please confirm.

Reply:
I misunderstood the key set in json. :scream:
Thanks for pointing that out.
https://github.com/yoshikawa-akr/js-lessons/pull/12/commits/c07f0a8b3bda147db9075f2c7af9dff067b77f61